lib/handler/RetryHandler.js

```diff
@@ -224,7 +230,7 @@ class RetryHandler {
       const { start, size, end = size } = contentRange
```

Comment:
@KhafraDev 
Here you see start, size and then end = size, which makes only sense in the unsatisfied case. But original parseRangeHeader can NOT return unsatisfied cases (e.g. `bytes */2222`). So this makes no sense. 

originally we only have start and end.